docs/data/joy/components/tabs/TabsBottomNavExample.js

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import Box from '@mui/joy/Box';
+import ListItemDecorator from '@mui/joy/ListItemDecorator';
+import Tabs from '@mui/joy/Tabs';
+import TabList from '@mui/joy/TabList';
+import Tab, { tabClasses } from '@mui/joy/Tab';
+import HomeOutlined from '@mui/icons-material/HomeOutlined';
+import FavoriteBorder from '@mui/icons-material/FavoriteBorder';
+import Search from '@mui/icons-material/Search';
+import Person from '@mui/icons-material/Person';
+
+export default function TabsBasic() {
+  const [index, setIndex] = React.useState(0);
+  const colors = ['primary', 'info', 'danger', 'success'];
+  return (
+    <Box
+      sx={{
+        flexGrow: 1,
+        m: -3,
+        p: 3,
+        py: 5,
+        borderRadius: 'sm',
+        bgcolor: `${colors[index]}.400`,
+      }}
+    >
```

Comment:
@siriwatknp 
I don't understand why in the docs often the sx syntax is used over the shorter:

```tsx
    <Box
      flexGrow={1}
      m={-3}
      p={3}
      py={5}
      borderRadius="sm"
      bgcolor=`${colors[index]}.400`
    >
```

Reply:
I rather use `sx` because I can copy and paste to another component when I have to refactor/move styles.

In the end, it depends on your preference.

@oliviertassinari I remember that you touch on this topic, is it?